db/migrate/20200422121348_add_channel_to_contacts.rb

```diff
@@ -0,0 +1,5 @@
+class AddChannelToContacts < ActiveRecord::Migration[6.0]
+  def change
+    add_column :contacts, :channel, :string
```

Comment:
I wonder if we should have a default here - i.e. `bulk import`

Reply:
yes good one

Reply:
or shall we call it "shield import"?